src/mcp_gateway/__main__.py

```
@@ -1,31 +1,24 @@
-    except Exception as e:
-        audit = AuditLogger()
-        audit.log(
-            ev="startup_failure",
-            level="ERROR",
-            error_type=e.__class__.__name__,
-            error=str(e),
-            stacktrace=traceback.format_exc(),
+        uvicorn.run(
+            "mcp_gateway.app:build_app", factory=True, host=host, port=port, log_level="info"
         )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)